core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournal.java

```diff
@@ -187,6 +198,7 @@ public URI getLocation() {
   @VisibleForTesting
   synchronized void write(JournalEntry entry) throws IOException, JournalClosedException {
     writer().write(entry);
+    mEntriesSinceLastCheckPoint++;
```

Comment:
will this every need to be an atomic increment? (It probably doesn't matter that much?)

Reply:
i am guessing not since the method is synchronized?